config/locales/en.yml

```diff
@@ -18,6 +18,10 @@ en:
     requests_for_extra_mobile_data: Requests for extra mobile data
     suggested_email_to_schools: A suggested email for you to send to schools
     request_extra_mobile_data: Request extra mobile data
+    request_extra_data_for_mobile_devices: Request extra data for mobile devices
+    how_would_you_like_to_submit_information: How would you like to submit information?
+    upload_a_spreadsheet_of_extra_data_requests: Upload a spreadsheet of extra data requests
+    weve_processed_your_spreadsheet: We've processed your spreadsheet
```

Comment:
```suggestion
    weve_processed_your_spreadsheet: We’ve processed your spreadsheet
```

Reply:
This is also a bit weird: ![Screen Shot 2020-08-03 at 16 56 21](https://user-images.githubusercontent.com/319055/89202106-5ecef700-d5aa-11ea-819b-ac01967c3e4b.png)

Reply:
Changing to single backquote has fixed this weird one.